docs/changelog.md

```diff
@@ -861,7 +861,7 @@ description: |
   - Import autocompletion now only shows the latest package version until a
     colon is typed
   - Fixed autocompletion for dictionary key containing a space
-  - Fixed autocompletion for for loops
+  - Fixed autocompletion for for-loops
```

Comment:
this `-` is to avoid the rule that doubled words are usually mistakes. Another approach would be `for` loops (and given that markdown is used below, that's probably better.

Reply:
do we necessarily need to satisfy that rule?

Reply:
Nope, I can add exceptions. The rule is mostly about making it easier to read. I think:

>  - Fixed autocompletion for `for` loops

would be easier for someone to read.